### DISCUSSION TITLE
Feature Request: Add Alias receiving the email to Sender Address Format

### DISCUSSION THREAD

**Discussion by Universal-Igloo** [simple-login/app#2207] [Feature Requests]
It often takes me a while to figure out which email was sent to which alias. Sometimes, the same email is sent to 2 of my aliases at the same time and I want to be specific as to which one to respond from, and the "To or CC" section in the email inbox doesn't help in that case.

Currently, there are 4 sender address formats to choose from.
1. John Wick - john(a)wick.com
2. John Wick
3. john at wick.com
4. No Name (i.e. only reverse-alias)

I recommend adding a 5th option, like:
5. John Wick - john(a)wick.com (to alias01@simplelogin.co)
Or something similar.

**Comment by nguyenkims** [comment]
When an email is sent to 2 aliases, there will be 2 address in the TO header:
- an alias and 
- a reverse alias for the 2nd alias

This should provide you the info you need right?

**Reply by Universal-Igloo** [reply]
Hi! Thank you for responding.

That doesn't exactly provide the information in a clear and reliable manner. Users still have to put in the guesswork as to whether the one listed as an alias or the one listed as a reverse alias is the primary recipient for that email.

If you are able to add a way to highlight the same very clearly in the email it would be very helpful. Two ways to acccomplish this:

1. Adding a 5th sender format similar to "John Wick - john(a)wick.com (to alias01@simplelogin.co)"
2. The way Anonaddy does it at the top of each email (see screenshot below)

# The way anonaddy does it...
<img width="720" alt="Screenshot 2024-09-09 at 17 34 25" src="https://github.com/user-attachments/assets/00ff916d-ae3e-446b-b53b-5ac00c1fe59a">

**Reply by nguyenkims** [reply]
Having an email address in the From header that doesn't match the real sender can be flagged by some email services as a phishing attempt so option 1 can't be added.

For option 2, we don't want to modify the email body as it'll be quoted automatically when you reply and so you'd need to manually remove this section.

**Reply by Universal-Igloo** [reply]
Hmm, all that makes sense of course. Thank you for looking into it.

How do you see a solution playing out for this? Is there any other option or is the current state the best there is?

Also, just to clarify, currently in the TO header if there is an "alias" (alias01@simplelogin.co) then that is the main recipient email, and if there is an alias in the reverse alias format then that is the one CC'ed, correct?

**Reply by winklerrr** [reply]
This problem seems to be related to my post https://github.com/simple-login/app/discussions/2426 . I added some possible solutions there. Could you please have a look?
